qcodes/instrument_drivers/tektronix/Keithley_2600_channels.py

```diff
@@ -338,6 +535,17 @@ def _fast_sweep(self, start: float, stop: float, steps: int,
 
         return outdata
 
+    def _check_autorange_v(self, val: int) -> None:
```

Comment:
This is surely a detail, but I find the name of this function confusing; it does not check anything. Consistently with other drivers, I think we should call it `_set_sourcerange_i`.

Reply:
Why isn't there a similar function for the measurerange?

Reply:
> This is surely a detail

in my book it is a very important detail :) i support this comment.